posthog/session_recordings/sql/session_replay_event_migrations_sql.py

```diff
@@ -114,3 +114,15 @@
     table_name=SESSION_REPLAY_EVENTS_DATA_TABLE(),
     cluster=settings.CLICKHOUSE_CLUSTER,
 )
+
+ALTER_SESSION_REPLAY_ADD_SOURCE_COLUMN_REMOVE_LOW_CARDINALITY = """
+    ALTER TABLE {table_name} on CLUSTER '{cluster}'
+    MODIFY COLUMN snapshot_source AggregateFunction(argMin, Nullable(String), DateTime64(6, 'UTC'))
+"""
+
+ALTER_SOURCE_SESSION_REPLAY_EVENTS_TABLE_SQL = (
```

Comment:
runs locally 👍 
same number of sessions before as after 👍 

leaves me with tables in this state

<img width="988" alt="Screenshot 2023-12-21 at 15 46 50" src="https://github.com/PostHog/posthog/assets/984817/5b5df979-07e0-4914-a22c-3f97b87ea5b1">